packages/fast-components-styles-msft/src/design-system/index.ts

```diff
@@ -109,6 +141,37 @@ export function ensureDesignSystemDefaults<T>(
     };
 }
 
+export function applyCornerRadius(

```

Comment:
Can you make seperate files for these utilities? I wouldn't expect these to live in the `designSystem` file

Reply:
I was thinking of it as convenience methods on things from `designSystem`, but that makes sense. I'll move them out.